### PR TITLE
Check hash of verified images after reboot

### PR DESCRIPTION
**Pull request by giggsoff**
Seems reasonable to check sha of file exists in verified directory
before assume that they are verified.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/pillar/cmd/verifier/verifier.go [hunk 1]
```diff
@@ -189,8 +189,25 @@ func handleInit(ctx *verifierContext) {
 
 	log.Functionln("handleInit")
 
-	// Create VerifyImageStatus for objects that were verified before reboot
-	handleInitVerifiedObjects(ctx)
+	// Run a periodic timer so we always update StillRunning
+	stillRunning := time.NewTicker(25 * time.Second)
```
Comment (eriknordmark) [inline pkg/pillar/cmd/verifier/verifier.go:193]: Makes sense to add a comment that the init reverification of the shas can take minutes for large objects.

File: pkg/pillar/cmd/verifier/verifier.go [hunk 2]
```diff
@@ -189,8 +189,25 @@ func handleInit(ctx *verifierContext) {
...
+	waitForVerifiedObjectsChan := make(chan bool, 1)
+	go func() {
+		// Create VerifyImageStatus for objects that were verified before reboot
+		handleInitVerifiedObjects(ctx)
+		waitForVerifiedObjectsChan <- true
+	}()
+
+	objectsVerified := false
+	for !objectsVerified {
+		select {
```
Comment (eriknordmark) [inline pkg/pillar/cmd/verifier/verifier.go:204]: Would it make sense to add a log.Notice that we are waiting for the initial objects to be verified?

File: pkg/pillar/cmd/verifier/verifier.go (hunk 1 above)
Comment (giggsoff) [reply pkg/pillar/cmd/verifier/verifier.go:194]: Added

File: pkg/pillar/cmd/verifier/verifier.go (hunk 2 above)
Comment (giggsoff) [reply pkg/pillar/cmd/verifier/verifier.go:205]: Added, and also for the end of the verification

**Review by rvs — COMMENTED** [review]
LGTM, but curious how much extra times this could consume and whether it would be noticeable @giggsoff

**Comment by giggsoff** [comment]
> LGTM, but curious how much extra times this could consume and whether it would be noticeable @giggsoff

If things goes well, we will not have any data inside verified directory, we will create blobs in containerd and remove content of verified directory. But If we will reboot during containerd blobs creation we assume that all data inside verified directory is correct. Here I put another run after non-usual reboot to check consistency of data.

**Pull request merged**